boxoffice/templates/boxoffice.js

```diff
@@ -888,5 +888,17 @@ $(function() {
   window.Boxoffice.init = function(widgetConfig) {
 	  boxoffice.init(widgetConfig);
   };
+
+  // Raise a custom event once boxoffice.js has been loaded
+  if (typeof(window.Event) === "function") {
+    var event = new Event('boxofficeScript-load');
```

Comment:
Can we use `on*` as a convention when triggering events. So, that would be `onBoxofficeInit` as the event name and `onBoxofficeInit` as the variable name.

Reply:
Fixed